inference-engine/src/plaidml_plugin/ops/reorg_yolo.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset4.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+static OpRegistration reg("reorgYolo", [](const Context& ctx) {
+  auto* layer = ngraph::as_type<ngraph::opset4::ReorgYolo>(ctx.layer);
+  IE_ASSERT(ctx.operands.size() == 1);
```

Comment:
Not sure if we shall check dimension/stride here, as openvino says "H and W should be divisible by stride"

Reply:
yes, I think check this rule here, which is very nice idea, I will add this.